### PR TITLE
update some git dep version tags that were failing, and run yarn upgrade

### PR DESCRIPTION
**Pull request by ohnorobo**
`yarn gulp build` started failing to find some dependencies, probably because of a fetching change between npmv6 and 7. (We use yarn, but npm is used by cordova.)
This slightly changes how those dependencies are specified to build again.

Jon and Ben, could you check out this branch and make sure it builds for you before it is merged?

### CODE REVIEW

**Comment by JonathanDCohen** [comment]
This builds for me on my mac.  Do you want to wait for review of your doc before proceeding?

**Comment by ohnorobo** [comment]
This is unrelated to the tradeoffs in go/outline-inter-repo-ts-dependencies, although both of them involve touching the version tags.
This is just about fixing the build failures that Ben and I were starting to see in outline-client.

File: package.json [hunk 1]
```diff
@@ -127,4 +127,4 @@
     "trailingComma": "es5",
     "printWidth": 120
   }
-}
+}
```
Comment (JonathanDCohen) [inline package.json:130]: Nit: trailing newline

**Review by JonathanDCohen — COMMENTED** [review]
Confirmed on npm 6 and 7 on mac, but I can't repro the original bug on my machine so @bemasc should double check

**Comment by bemasc** [comment]
Confirmed, this branch fixes my build error.

File: package.json (hunk 1 above)
Comment (ohnorobo) [reply package.json:130]: fixed

**Comment by ohnorobo** [comment]
After rebasing and reapplying this change I'm now seeing the build fail on

ERROR in Error: The loader "[...]outline-client/src/www/cordova_index.html" didn't return html.

I'm assuming this is failing because of some bad package interaction in the yarn.lock file. Doesn't seem worth debugging, so for the moment I'm going to wait a few days and see if a new `yarn upgrade` fixes things.

**Comment by ohnorobo** [comment]
Bug seems potentially related to https://github.com/webpack/webpack/issues/12687, which can be fixed by upgrading html-webpack-plugin

File: package.json (hunk 1 above)
Comment (ohnorobo) [reply package.json:130]: un-fixed. Turns out this newline is coming from the presubmit formatter, so we should just let it be.

**Comment by ohnorobo** [comment]
Got the build working again by also upgrading html-webpack-plugin.

**Review by JonathanDCohen — APPROVED** [review]

**Pull request merged**